packages/pangea-node-sdk/tsconfig.json

```diff
@@ -19,6 +19,9 @@
     "noUncheckedIndexedAccess": true,
     "noEmitOnError": true,
     "useDefineForClassFields": true,
+    "target": "es5",
```

Comment:
Why?

Reply:
Without a target, we get a lot of compiler warnings. ES5 works both in browser and node for the most part.

Reply:
Removed